Makefile

```diff
@@ -0,0 +1,53 @@
+PYTHON = $(shell pyenv version)
+MANAGE = $(PYTHON) manage.py
+ENV = $(CURDIR)/env
+PYTHON = $(ENV)/bin/python3
+COVERAGE = $(ENV)/bin/coverage
+COVERAGE_OPTS = --rcfile=coverage.cfg
+
+all: start
+
+start run: test
+	$(MANAGE) runserver 0.0.0.0:8000
+
+test: $(ENV)/bin/nose2 $(ENV)/bin/coverage
+	$(MANAGE) test
+
+$(ENV)/bin/nose2 $(ENV)/bin/coverage: requirements
+
+requirements: migrate
```

Comment:
Yeah, it's there. It's just a requirements.txt file in the parent folder of the project. I don't pin a version, which we probably should, but coverage is the last listed requirement.